### PR TITLE
Operator late customization management (bug 1205496)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1205496

late-customization-manage-2 mov

Changes are saved automatically. This isn't the prettiest code I've written.

### CODE REVIEW

**Comment by mstriemer** [comment]
This depends on mozilla/zamboni#3335.

File: src/media/js/views/late-customizations.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        $form.find('select').removeAttr('disabled');
+        $form.find('.manage-late-customizations').empty();
+    }
+
+    function manageForm($form) {
+        var data = getFormData();
+        cache.flush();
+        requests.get(urls.api.url('late-customizations', [], data))
+                .then(function(response) {
+            Promise.all(response.objects.map(function(setApp) {
```
Comment (mstriemer) [inline src/media/js/views/late-customizations.js]: I'll need to polyfill this or figure out how to make this work with `defer.when` but that didn't seem to want to work before.

File: src/media/js/main.js [hunk 2]
```diff
@@ -49,7 +49,8 @@ require(
 
     z.page.on('loaded logged_in', function() {
         if (user.logged_in() && !user.get_permission('curator') &&
-            !user.get_permission('admin')) {
+            !user.get_permission('admin') &&
+            utils.baseurl(location.pathname) !== urls.reverse('403')) {
```
Comment (mstriemer) [inline src/media/js/main.js]: This is unrelated but prevents the site from redirecting to `/403` if you're already on the `/403` page which locks up the browser.

**Comment by ngokevin** [comment]
r+

**Comment by ngokevin** [comment]
wait isn't this supposed to be on the operator dashboard?

**Comment by mstriemer** [comment]
Closing in favour of https://github.com/mozilla/marketplace-operator-dashboard/pull/29.

**Pull request closed without merging**